tests/Bech32AssetTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace CardanoPhp\Bech32;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+#[CoversClass(Bech32::class)]
+class Bech32AssetTest extends TestCase
```

Comment:
Minor recommendations:

* Expand test coverage with unhappy path also
  * Add test that provide invalid input and edge cases (min/max input, zero lonth and other boundary conditions)
  * Use php unit's `assertException` helper to check expected exceptions are thrown for invalid inputs / edge cases